statsmodels/stats/proportion.py

```diff
@@ -112,14 +112,16 @@ def func(qi):
         ci_low = center - dist
         ci_upp = center + dist
 
-    elif method == 'jeffrey':
-        ci_low, ci_upp = stats.beta.interval(1 - alpha,  count + 0.5,
-                                             nobs - count + 0.5)
+    # method adjusted to be more forgiving of misspellings or incorrect option name
+    elif method[:4] == 'jeff':
+            ci_low, ci_upp = stats.beta.interval(1 - alpha, count + 0.5,
```

Comment:
You are correct, I had an extra 4 spaces in there.  I must have hit tab accidentally.